src/js/views/graph-view.js

```diff
@@ -362,8 +362,14 @@ var graphView = {
       if ( SSData.past_fra ) {
         percent = ( SSData['age' + SSData.currentAge] - SSData['age' + this.selectedAge] ) / SSData['age' + SSData.currentAge];
         var comparedToClaimingFullEs = window.gettext( 'Compared to claiming at' );
-        var comparedToClaimingEs = comparedToClaimingFullEs.split( 'XXX' );
-        $( '.compared-to-full' ).html( comparedToClaimingEs[0] + SSData.fullAge + comparedToClaimingEs[1] );
+        var comparedToClaimingEsSplit = comparedToClaimingFullEs.split( 'XXX' );
+        if ( typeof comparedToClaimingEsSplit !== 'undefined' ) {
+          if ( comparedToClaimingEsSplit.length == 2 ) {
```

Comment:
Should that be `comparedToClaimingEsSplit.length === 2` (strict equality operator)?